curriculum/challenges/english/01-responsive-web-design/basic-css-cafe-menu/part-011.md

```diff
@@ -11,10 +11,22 @@ Up until now, you have been limited regarding the presentation and appearance of
 
 # --hints--
 
-Test 1
+Your code should have an opening `<style>` tag.
 
 ```js
+assert(code.match(/<style>/i));
+```
+
+Your code should have a closing `</style>` tag.
 
+```js
+assert(code.match(/<\/style>/));
+```
+
+Your `style` element should be nested in your `head` element.
+
+```js
+assert(code.match(/<head>[\w\W\s]*<style>[\w\W\s]*<\/style>[\w\W\s]*<\/head>/i))
```

Comment:
@ojeytonwilliams At one point you were going to attempt to fix the issue where we can not easily query the head and style elements without having to use regex to find them.

Reply:
I did look into that.  As I recall, the problem is that browsers automatically create elements and re-order others, so you can't always know what the user did.  It's been a while, but I think `<title>`s get put into `<head>`s because that's the only permitted parent.  That sort of thing.

We can probably do better than our current approach, but I think there will always be oddities.